### DISCUSSION TITLE
CI not starting

### DISCUSSION THREAD

**Discussion by Xwilarg** [community/community#27197] [Actions]
<p>Hi,</p>
<p>So I’m trying to introduce someone to C++ (and GitHub) so I showed him some basic stuffs, along with GitHub Action<br>
He made this configuration file: <a href="https://github.com/loge0/generation/blob/3900767a408aac9144dba2a26981081448424496/.github/workflows/windows.yml" rel="noopener nofollow ugc">https://github.com/loge0/generation/blob/3900767a408aac9144dba2a26981081448424496/.github/workflows/windows.yml</a><br>
But even when he push, the “Actions” tab stays empty: <a href="https://github.com/loge0/generation/actions" rel="noopener nofollow ugc">https://github.com/loge0/generation/actions</a></p>
<p>Would someone please know why this is happening?</p>
<p>Thanks in advance,</p>

**Comment by kingthorin** [comment, marked as answer]
<p>The workflow says “master” but default branch in the repo is “main”.</p>

**Comment by Xwilarg** [comment]
<p>Oh I forgot about these new changes, thanks!</p>
